src/index.html

```diff
@@ -11,8 +14,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Grey monitor iMac product photo"
+        class="card__image"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <span class="card__subtitle">Product code: 195434</span>
+
+      <section class="card__rating">
+        <div class="card__stars stars stars--4">
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+        </div>
+
+        <span class="card__reviews-label">Reviews: 5</span>
+      </section>
+
+      <section class="card__price">
+        <span class="card__price-label">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </section>
+
+      <a
+        href="#"
+        data-qa="hover"
+        class="card__button card__button--styles"
+      >
+        BUY
+      </a>
```

Comment:
The text inside the anchor tag should be in a '<span>' tag to provide more styling options and better accessibility.